### ISSUE TITLE
Accessibility: ready to review queue should have describedby pointing to descriptions

### ISSUE THREAD

**Issue by alavatelliMCA** [coyote-team/coyote#8]
ready to review queue should have describedby pointing to descriptions

**Comment by seeReadCode** [comment]
so the actions per each description in the ready to review now have a described by pointing to the description
